src/clients/gists.js

```diff
@@ -1,25 +1,10 @@
-import assign from 'lodash/assign';
```

Comment:
Combined `clients`, `repos`, and `services/github` into a single `clients/github` module